rust/src/api/lbtc_ln.rs

```diff
@@ -409,7 +413,7 @@ impl LbtcLnV2Swap {
             from: "L-BTC".to_string(),
             to: "BTC".to_string(),
             invoice: invoice.to_string(),
-            referral_id: None,
+            referral_id: referral_id.clone(),
```

Comment:
i dont think we need to .clone() we can just consume it since it not being used later in the flow

Reply:
i was using it on line 440

Compiler complains.

I was returning the value of the referral to user 🤔

Reply:
ahh, yes, sorry!

Its part of `*LnV2Swap` now